portfolio/src/main/java/com/google/DataServlet.java

```diff
@@ -11,22 +11,33 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps.servlets;
-
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+ 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/data") 
 public class DataServlet extends HttpServlet {
-
+  private List<String> phrases;
+   @Override
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.1-vertical-whitespace
Put a line between each of your member variables and functions.

{
  private List ...

  @Override
  public void init() {
  }

  @Override
  public void doGet

Reply:
Thank you for approving, I have implemented this change for the next PR (not the jsonBranch but the postBranch that I will send today).